modules/cloudsql-postgres/main.tf

```diff
@@ -129,6 +139,8 @@ resource "google_sql_database_instance" "replicas" {
       settings[0].disk_size
     ]
   }
+
+  depends_on = [google_sql_database_instance.this]
```

Comment:
i like the depends_on but you dont need that as you make a dependency when you link that in the line 105